app/templating/schema_context.py

```diff
@@ -1,6 +1,7 @@
-from app import settings
 from app.utilities.date_utils import to_date
 
+EQ_MAX_NUM_REPEATS = 25
```

Comment:
Not ideal having this same value defined in multiple places, what happens if they get out of sync...